data/genesis/demo-epoch.toml

```diff
@@ -0,0 +1,33 @@
+base_version = "0.2"
+upgrade_version = "0.3"
+
+[stake_table]
+capacity = 10
+
+[chain_config]
+chain_id = 999999999
+base_fee = '0 wei'
+max_block_size = '1mb'
+fee_recipient = '0x0000000000000000000000000000000000000000'
+fee_contract = '0xa15bb66138824a1c7167f5e85b957d04dd34e468'
+
+[header]
+timestamp = "1970-01-01T00:00:00Z"
+
+[l1_finalized]
+number = 0
+
+[[upgrade]]
+version = "0.99"
```

Comment:
@imabdulbasit Can you remind me why this is 0.99 and not 0.3?

Reply:
fixed this locally but this should be v3